### PR TITLE
[EN] osu!wiki_contribution_guide update v2

### PR DESCRIPTION
**Pull request by RaikireHiuduo**
Changes:-
- Added rename/moving and delete file in GitHub web editor.
- Removed the redirect problem in FAQ section.
- Add new branch own upstream set.
- ~~Are we still using the 639-1 code? What is the new status?~~
  - Rewrote to base on #349 new locales section in ASG.
- Random formatting and text changes.
 
P.S. This branch was made inside the Visual Studio Code's integrated Git (need to download and install ``git-scm`` first).

### CODE REVIEW

File: wiki/osu!wiki_contribution_guide/en.md [hunk 1]
```diff
@@ -57,6 +63,8 @@ For a styling guide when writing a page, check out [Article Style Guide][ASG] in
 
 If you have any questions, you can inquire at the [osu!dev Discord Server][osu!dev Discord], under ``#osu-wiki`` preferably.
 
+Frequently-Asked-Questions (FAQ) for osu!wiki development in GitHub can be found at the last header of the page, namely [FAQ][FAQ].
```
Comment (oralekin) [inline wiki/osu!wiki_contribution_guide/en.md]: `last header of this page`

File: wiki/osu!wiki_contribution_guide/test.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+Addition, deletion, moving, and renaming of media is easier now and requires no special wiki function.
+However, this too has to undergo the pull request protocol.
+While osu!wiki will resize images to the width size limit if the image is too large, specific media size requirements must be handled manually.
+
+The most damaging change for pages that use it liberally would be **no Templates support**.
+You must manually duplicate the template and paste it accordingly with proper changes to the article you are editing, _on every language file_.
+
+Tables are supported by a bare-minimum with GFM (without extensions), unlike MediaWiki.
+
+Markdown is sensitive, as using the wrong capitalization in links can break them (i.e. links and images).
```
Comment (kj415j45) [inline wiki/osu!wiki_contribution_guide/test.md]: ASG: i.e. -> e.g.

File: wiki/osu!wiki_contribution_guide/test.md [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+### Offline editors
+
+Want to take your work offline?
+No problem!
+
+Open up your favourite text-editor like Notepad/gedit/Text Document and save the new file with an extension name of ``.md`` at the end of the filename.
+Then, copy the content from the GitHub Web Editor and paste it into your currently-opened file.
+Make your changes and do it in reverse and hit the update button.
+
+If you want a GFM previewer or syntax highlighting, the osu!wiki team recommends:-
```
Comment (kj415j45) [inline wiki/osu!wiki_contribution_guide/test.md]: ~~MegaApplePi's [mdp-osu](https://github.com/MegaApplePi/mdp-osu)~~

File: wiki/osu!wiki_contribution_guide/test.md [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+## FAQ
+
+### But Markdown supports HTML tags, can I use them?
+
+**No.**
+This also includes any CSS styling, inline styling and colours.
+
+The current osu!wiki supports **GFM syntax only**.
+
+GFM is based on Markdown; which is basically a simplified subset of important HTML tags for styles that contains:
```
Comment (kj415j45) [inline wiki/osu!wiki_contribution_guide/test.md]: how about `<br/>`:
double space character at the end of line.

File: wiki/osu!wiki_contribution_guide/test.md [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+
+If you want a GFM previewer or syntax highlighting, the osu!wiki team recommends:-
+
+- Microsoft's [Visual Studio Code][VSC],
+- GitHub's [Atom][Atom], or
+- Adobe's [Brackets][Brackets].
+
+The osu!wiki team also recommend [Shiba][Shiba] for checking GFM syntax compliance and page preview.
+
+---
```
Comment (kj415j45) [inline wiki/osu!wiki_contribution_guide/test.md]: `### Advanced Guide`
(?)

File: wiki/osu!wiki_contribution_guide/test.md [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+- [Git Extensions][Git Extensions]
+  - [Git Extension User Manual][Git Extension User Manual]
+- Microsoft's [Visual Studio Code][VSC] (requires external installation of git-scm to work)
+
+### Offline editors
+
+Want to take your work offline?
+No problem!
+
+Open up your favourite text-editor like Notepad/gedit/Text Document and save the new file with an extension name of ``.md`` at the end of the filename.
```
Comment (kj415j45) [inline wiki/osu!wiki_contribution_guide/test.md]: **Never `Notepad`!**

**Comment by TPGPL** [comment]
I would delete the FAQ header or move it to /wiki/FAQ

**Comment by RaikireHiuduo** [comment]
@TPGPL : The FAQ is not even related to the _osu!_ game itself; its for the wiki (there is only like one or two notes that are still relevant anyway).

Regardless, closing this since I give up to #413 .

**Pull request closed without merging**